apps/etterlatte-behandling/src/main/kotlin/behandling/GrunnlagService.kt

```diff
@@ -33,4 +41,18 @@ class GrunnlagService(private val grunnlagKlient: GrunnlagKlientImpl) {
             persongalleri = persongalleri
         )
     }
+}
+
+internal fun <T : Any> lagOpplysning(
```

Comment:
Denne burde nok ligge i grunnlag-model. Ser også at `Opplysningsuthenter.kt` har denne som du sikkert har kopiert i fra, den kan i så fall bruke den samme.

Reply:
Tja, jeg er ikke helt enig. Jeg vil heller ha "duplikatkode" to steder, enn å opprette _enda et lib_. Finnes p.t. ingen `grunnlag-modell` og vi har ikke akkurat mangel på libs som kun sparer oss for noen få linjer kode og gir oss masse unødvendig overhead.